### PR TITLE
Add footer link component

### PR DESCRIPTION
**Pull request by santoshdasa12345**
## Description of change

Create Footer Link for Reminder and settings Page

## Test instructions

Click notifications--> Click on export companies with new interactions--> Footer -->guidance on reminders and email notifications link
Click notifications--> Click on Investment Project with no recent interactions--> Footer -->guidance on reminders and email notifications link

## Screenshots

image
image
image
image
image
image

### After
image
image
image
image
image
image
image

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'

- [x] Has the branch been rebased to main?
- [x] Automated tests (Any of the following when applicable: Unit, Functional or End-to-End)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, Edge, Safari)

### CODE REVIEW

File: src/client/modules/Reminders/FooterLink.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+})
+
+const FooterLink = ({ headingText, linkText, linkUrl }) => {
+  return (
+    <StyledFooterLink>
+      <div data-test="footer-link">
+        <hr />
+        <h2>{headingText}</h2>
+        <p>
+          See the <a href={linkUrl}>{linkText}</a>
```
Comment (chopkinsmade) [inline src/client/modules/Reminders/FooterLink.jsx:22]: You can use this  `import { Link } from 'govuk-react'` instead of a manual created `<a>`

Comment (chopkinsmade) [inline src/client/modules/Reminders/FooterLink.jsx:22]: Should this link open in a new tab? Currently this takes you away from datahub in your current tab

File: src/client/modules/Reminders/Settings/ExportsNewInteractionsForm.jsx [hunk 2]
```diff
@@ -23,37 +24,44 @@ const whenYouWantQuestion =
...
+              doYouWantQuestion={doYouWantQuestion}
+              whenYouWantQuestion={whenYouWantQuestion}
+              remindersDaysLabelHint={remindersDaysLabelHint}
+              emptyErrorMessage="Add when you want to get company reminders"
+            />
+          )}
+        </>
+      )}
+    </Form>
+    <FooterLink
```
Comment (chopkinsmade) [inline src/client/modules/Reminders/Settings/ExportsNewInteractionsForm.jsx:59]: If you move this `<FooterLink/>` usage to the file `src/client/modules/Reminders/Settings/RemindersForms.jsx` you can avoid duplicating the component in this file and the other 3 form files it's been added to

File: src/client/modules/Reminders/FooterLink.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    marginLeft: 1,
+    marginBottom: 25,
+  },
+})
+
+const FooterLink = ({ headingText, linkText, linkUrl }) => {
+  return (
+    <StyledFooterLink>
+      <div data-test="footer-link">
+        <hr />
```
Comment (chopkinsmade) [inline src/client/modules/Reminders/FooterLink.jsx:19]: If you look at the after screenshots, the bottom of the existing content has an `<hr/>` and this component also renders a `<hr/>` so it looks a bit strange. Do the designs cater for this?

File: src/client/modules/Reminders/Settings/ExportsNewInteractionsForm.jsx (hunk 2 above)
Comment (santoshdasa12345) [reply src/client/modules/Reminders/Settings/ExportsNewInteractionsForm.jsx:59]: resolved

File: src/client/modules/Reminders/FooterLink.jsx (hunk 1 above)
Comment (santoshdasa12345) [reply src/client/modules/Reminders/FooterLink.jsx:22]: resolved

Comment (santoshdasa12345) [reply src/client/modules/Reminders/FooterLink.jsx]: resolved

File: src/client/modules/Reminders/FooterLink.jsx (hunk 3 above)
Comment (santoshdasa12345) [reply src/client/modules/Reminders/FooterLink.jsx]: resolved

File: src/client/modules/Reminders/FooterLink.jsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    marginLeft: 1,
+    marginBottom: 25,
+  },
+})
+
+const FooterLink = ({ headingText, linkText, linkUrl }) => {
+  return (
+    <StyledFooterLink>
+      <div data-test="footer-link">
+        <h2>
```
Comment (chopkinsmade) [inline src/client/modules/Reminders/FooterLink.jsx:20]: I've just found this, there is a H2 component in the govuk-react package: `import { H2 } from '@govuk-react/heading'`

File: src/client/modules/Reminders/Settings/ExportsNewInteractionsForm.jsx [hunk 5]
```diff
@@ -23,37 +23,39 @@ const whenYouWantQuestion =
...
-            doYouWantQuestion={doYouWantQuestion}
-            whenYouWantQuestion={whenYouWantQuestion}
-            remindersDaysLabelHint={remindersDaysLabelHint}
-            emptyErrorMessage="Add when you want to get company reminders"
-          />
-        )}
-      </>
-    )}
-  </Form>
+  <>
```
Comment (chopkinsmade) [inline src/client/modules/Reminders/Settings/ExportsNewInteractionsForm.jsx:26]: You don't need the empty <> anymore, same applies to the next few components

File: src/client/modules/Reminders/Settings/RemindersForms.jsx [hunk 6]
```diff
@@ -16,50 +16,58 @@ import InvestmentsEstimatedLandDatesForm from './InvestmentsEstimatedLandDatesFo
...
-        )}
-        {reminderType === COMPANIES_NO_RECENT_INTERACTIONS && (
-          <ExportsNoRecentInteractionsForm />
-        )}
-        {reminderType === COMPANIES_NEW_INTERACTIONS && (
-          <ExportsNewInteractionsForm />
-        )}
-      </>
-    </DefaultLayout>
+    <>
```
Comment (chopkinsmade) [inline src/client/modules/Reminders/Settings/RemindersForms.jsx:27]: You can remove these `<>` as well, all the content sits within the `<DefaultLayout>` component

Comment (santoshdasa12345) [reply src/client/modules/Reminders/Settings/RemindersForms.jsx]: resolved

File: src/client/modules/Reminders/Settings/ExportsNewInteractionsForm.jsx (hunk 5 above)
Comment (santoshdasa12345) [reply src/client/modules/Reminders/Settings/ExportsNewInteractionsForm.jsx]: resolved

File: src/client/modules/Reminders/FooterLink.jsx (hunk 4 above)
Comment (santoshdasa12345) [reply src/client/modules/Reminders/FooterLink.jsx]: resolved

**Review by chopkinsmade — APPROVED** [review]

**Review by dredmonds — APPROVED** [review]

**Pull request merged**